builder/tests/builder/src/main/java/io/helidon/builder/test/testsubjects/ContainerBlueprint.java

```diff
@@ -18,30 +18,14 @@
 
 import java.util.Optional;
 
-import io.helidon.builder.Builder;
+import io.helidon.builder.api.Prototype;
 import io.helidon.config.metadata.ConfiguredOption;
 
 /**
  * Base for {@link PickleBarrel}.
  */
-@Builder
-public interface Container {
-
-    /**
-     * Type of container.
-     */
-    enum Type {
-        /**
-         * WOOD.
-         */
-        WOOD,
-
```

Comment:
are embedded type/enum definitions no longer supported??

Reply:
No, because the blueprint is required to be package local. they can still be embedded, but in a public type.